e2e/tests/room/09-jumptomessage.spec.js

```diff
@@ -0,0 +1,295 @@
+const {
+	device, expect, element, by, waitFor
+} = require('detox');
+const data = require('../../data');
+const { navigateToLogin, login, mockMessage, tapBack, sleep, searchRoom, starMessage, pinMessage, dismissReviewNag, tryTapping } = require('../../helpers/app');
+const { post, get } = require('../../helpers/data_setup');
+const deepLink = msg => `https://mobile.rocket.chat/channel/jumpToMessageChatrwurojjluzrbfxyocppa?msg=${msg}`;
```

Comment:
Use the static room and make sure to use the proper server.
Mobile vs docker

`data.server.*`